src/ui/proxy/interceptors.go

```diff
@@ -341,13 +340,17 @@ func copyResp(rec *httptest.ResponseRecorder, rw http.ResponseWriter) {
 	rw.Write(rec.Body.Bytes())
 }
 
-func marshalError(msg string, statusCode int) string {
-	je := &JSONError{
-		Message: msg,
-		Code:    statusCode,
-		Details: msg,
+func marshalError(msg string) string {
+	var tmpErrs struct {
+		Errors []JSONError `json:"errors,omitempty"`
 	}
-	str, err := json.Marshal(je)
+	tmpErrs.Errors = append(tmpErrs.Errors, JSONError{
+		Code:    "UNKNOWN",
```

Comment:
Would be better set some meaningful, even though it can't be handled by vanilla docker currently?

Reply:
How'bout PROJECT_POLICY_VIOLATION ?